### DISCUSSION TITLE
Daily 19/9

### DISCUSSION THREAD

**Discussion by rpgrca** [rpgrca/proyecto-ispc-fullstack#14] [General]
Qué es lo que planean hacer hoy? Ven alguna dificultad?

**Comment by rpgrca** [comment]
Yo voy a seguir puliendo el ERS, subimos una versión preliminar pero que tenemos que terminar en base a lo que escribamos en la Wiki. No creo que termine hoy así que voy a necesitar que me den una mano con eso.

**Comment by rocappellano** [comment]
Estaré practicando HTML, y repasando las historias de usuario

**Comment by Brian-Almada** [comment]
Recién termino de clonar el repositorio del proyecto y de leer la documentación, el WIKI y las demás secciones.
Lo siguiente será repasar ingles para dar el examen en la semana.
Luego, repasar algunos puntos de Python para dar el examen final esta semana.
Por último, completar la parte de "Propósito" que se encuentra en la sección "Introducción" de la especificación de requisitos de software.

La dificultad del día está en que para completar la última tarea tengo que ver la manera correcta de hacerlo en los tutoriales de IEEE830. Hasta que repase me va a llevar tiempo. Aún así, creo que voy a poder completarla.

**Comment by NataliaAlvarezIspc** [comment]
Estaré terminando de estudiar HTML. Lectura sobre requerimientos de IEEE830 y redacción adecuada de #US.
La dificultad puede estar en el tiempo que le pueda dedicar para llegar a hacer todo.

**Comment by AhumadaCeleste** [comment]
En el día de ayer, ínstela xampp. Tuve dificultades . las cuales pude resolver en el día de la fecha. 
Hoy clone el nuevo repositorio. Continuare  con el apunte de guía de html, acompañándolo de practica en Visual Studio Code.
Mañana estudiare inglés con el propósito de reforzar contenidos dado a que se encuentra habilitado el examen bloque 3.

**Comment by asm3456** [comment]
Estoy tratando de instalar, los programas si me estoy poniendo al día html ,también para clonar el repositorio

**Comment by Thiagox749** [comment]
El dia de ayer realicé un repaso de los módulos de Python.
Hoy empecé a leer los apuntes de CSS, aplicándolo junto con html para poner en practica.
Mañana repasaré inglés y clonare el repositorio.

**Comment by IngridAlcocer05** [comment]
Hoy repase el contenido de HTML y continúe con la lectura de la guía de CMS.
